go.mod

```diff
@@ -75,7 +75,7 @@ require (
 	github.com/prometheus/procfs v0.7.3 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
 	github.com/test-network-function/graphsolver-exports v0.0.0-20220909223441-46e11b864674 // indirect
-	github.com/test-network-function/l2discovery-exports v0.0.0-20220929081721-4e9a42510716 // indirect
+	github.com/test-network-function/l2discovery-exports v0.0.0-20220926190927-5f72a82c96d4 // indirect
```

Comment:
This should be upgraded to 
v0.0.0-20220926190927-5f72a82c96d4 => v0.0.0-20220929081721-4e9a42510716
or else it error's out 
 ptpInterface.IfPci.Description undefined

Reply:
need to update go mod vendor